src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/RedHatSeriesMenuCard/RedHatSeriesMenuCard.tsx

```diff
@@ -42,7 +42,7 @@ const RedHatSeriesMenuCard: FC<RedHatSeriesMenuCardProps> = ({
 
   const { classAnnotation, seriesName, sizes } = rhSeriesItem;
 
-  const { Icon, seriesLabel, seriesTitle } = instanceTypeSeriesNameMapper[seriesName];
+  const { Icon, seriesLabel, seriesTitle } = instanceTypeSeriesNameMapper?.[seriesName] || {};
```

Comment:
instanceTypeSeriesNameMapper[seriesName] || {} should be enough as instanceTypeSeriesNameMapper is a const.